### ISSUE TITLE
Fix pyflakes for pair

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#6993]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)| @adiroiban reported|
|-|-|
|Trac ID|trac#6993|
|Type|enhancement|
|Created|2014-02-27 12:31:33Z|
|Branch|https://github.com/twisted/twisted/tree/pyflakes-pair-6993|

Attachments:

* [6993-1.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/73a/73a5135c1a9bafa20c64829a43d6103c204d8e86/101ecd3d28cf03086a6579c636983615d7f38ad1.diff) (2878 bytes) - added by adiroiban on 2014-02-27 12:35:42Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__6993 6993
type__enhancement enhancement
reporter__adiroiban adiroiban
priority__normal normal
milestone__ 
branch__branches_pyflakes_pair_6993 branches/pyflakes-pair-6993
branch_author__exarkun exarkun
status__closed closed
resolution__fixed fixed
component__pair pair
keywords__ 
time__1393504293000000 1393504293000000
changetime__1393507037000000 1393507037000000
version__None None
owner__exarkun exarkun

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_1"></a>|@adiroiban commented|
|-|-|

Looks like some copy paste error for unused imports + other unused imports.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_3"></a>|@exarkun commented|
|-|-|

(In [41760]) Branching to 'pyflakes-pair-6993'

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_4"></a>|@exarkun commented|
|-|-|

(In [41762]) Apply 6993-1.diff 

refs [#6993](https://github.com/twisted/twisted/issues/6993)

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun commented|
|-|-|

Thanks.  Looks great to me.  Merging.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_6"></a>|@exarkun set status to `closed`|
|-|-|

(In [41765]) Merge pyflakes-pair-6993

Author: adiroiban
Reviewer: exarkun
Fixes: [#6993](https://github.com/twisted/twisted/issues/6993)

Remove unused imports from modules in Twisted Pair.  Twisted Pair is now
"pyflakes clean".
